web/src/components/auth/IRMAAuthentication.vue

```diff
@@ -32,28 +24,30 @@ export default {
         // Front-end options
         language: 'en',
         translations: {
-          header: "Authenticate for " + this.customer.name
+          header: this.headerMessage
         },
 
         // Back-end options
         session: {
-          // Point to demo-ehr backend which forwards requests
+          // Set base-url path to demo-ehr endpoint which forwards requests to IRMA server (embedded in the Nuts Node)
           url: '/web/auth/irma',
 
           // Define your disclosure request:
           start: {
+            url: o => `${o.url}/session`,
```

Comment:
isn't this the default?

Reply:
yes. Added for clearity. Had to look it up in the docs.